spec/support/helpers/hbw_fields_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Features
+  module HBWFieldsHelper
```

Comment:
This datetime picker is used in HOMS as well. Also, let it be FillHelper or DateTimeHelper.

Reply:
Done